pulsar/consumer.go

```diff
@@ -139,46 +141,15 @@ type Consumer interface {
 	// This calls blocks until a message is available.
 	Receive(context.Context) (Message, error)
 
-	// ReceiveAsync appends the message to the msgs channel asynchronously.
-	ReceiveAsync(ctx context.Context, msgs chan<- ConsumerMessage) error
-
-	// ReceiveAsyncWithCallback returns a callback containing the message and error objects
-	ReceiveAsyncWithCallback(ctx context.Context, callback func(msg Message, err error))
+	// Messages returns a channel to consumer messages from
+	Messages() <-chan ConsumerMessage
```

Comment:
Wouldn't be more intuitive to call this `Chan()` ?

Reply:
What about MessageChan() or Chan() works too?